src/layouts/RootLayout.tsx

```diff
@@ -0,0 +1,12 @@
+import { Outlet } from 'react-router-dom';
+
+/** App root layout with nested route outlet */
```

Comment:
For components lets use this type of documentation. Make sure we follow this on all the components/containers we create

```suggestion
/**
 * RootLayout component
 *
 * Provides the main application layout structure with a header, main content area (using React Router's Outlet for nested routes), and a footer.
 *
 * @component
 * @example usage
 * ```tsx
 * <Route path="/" element={<RootLayout />}>
 *  ...
 * </Route>
 * ```
 */
```